projects/storefrontapp/src/app/app.module.ts

```diff
@@ -51,7 +55,18 @@ if (environment.b2b) {
 
     ...devImports,
   ],
-
   bootstrap: [StorefrontComponent],
 })
-export class AppModule {}
+export class AppModule {
+  constructor(private x: EventService) {
```

Comment:
If you think you're done with testing the Corey's issue, let's clean up this `app.module`.
I think we can safely revert it.